kits/snapkart/apps/src/actions/utils.py

```
@@ -0,0 +1,81 @@
+        items: List of dicts with name, quantity, and unit fields
+        
+    Returns:
+        Formatted string like "2 piece surf excel chota wala, 1 kg chini"
+    """
+    return ", ".join(
+        f"{item.get('quantity', 1)} {item.get('unit', 'piece')} {item['name']}"
+        for item in items
+    )
+
+
+def classify_intent(message: str) -> str:
```

Ruff (0.15.21)

[warning] Unused function argument: `message`

(ARG001)